raiden/services.py

```diff
@@ -113,7 +113,8 @@ def update_monitoring_service_from_balance_proof(
             "Skipping update to Monitoring service. "
             "Your channel balance {channel_balance} is less than "
             "the required minimum balance of {min_balance} "
-            "that you have set before sending the MonitorRequest"
+            "that you have set before sending the MonitorRequest,"
+            " token address {token_address}"
```

Comment:
Can you make this a whole sentence? Like

"... before sending the MonitorRequest, " 
"in the Token Network {token_address}."